src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+
+  if (days < 3) {
+    totalAmount = days * 40;
+  } else if (days >= 3 && days < 7) {
+    totalAmount = days * 40 - 20;
+  } else {
+    totalAmount = days * 40 - 50;
+  }
```

Comment:
you don't need `days >= 3` in the second if

check your code with checklist 
<img width="1061" alt="Screenshot 2022-09-15 at 17 08 16" src="https://user-images.githubusercontent.com/37452763/190425487-96481a57-346b-4069-ab97-a2a3e627b5a8.png">